sql/2019/18_Page_Weight/18_01c.sql

```diff
@@ -0,0 +1,12 @@
+#standardSQL
+# 18_01c: Average size of each of the resource types.
+SELECT
+  ROUND(AVG(bytesTotal) / 1024) AS total_kbytes,
```

Comment:
Round to 2 decimal places, for each of these
```suggestion
  ROUND(AVG(bytesTotal) / 1024, 2) AS total_kbytes,
```